### ISSUE TITLE
Please tag a release and gpg sign

### ISSUE THREAD

**Issue by jonathancross** [litecoin-project/p2sh-convert#12]
Would be great to have an official release of this tool tagged with an OpenPGP signature.
This way we can have users download and check the signature before trusting (removing the need to trust GitHUb).

I still have some minor visual formatting I'd like to do (eg: make it responsive for mobile users), but I thought I'd get this conversation started as security is more important :-)

**Comment by losh11** [comment]
Agreed. Would be nice to have a folder with developer pubkeys with signature verification instructions in README.

**Comment by Veezywqp1** [comment]
> Would be great to have an official release of this tool tagged with an OpenPGP signature. This way we can have users download and check the signature before trusting (removing the need to trust GitHUb).
> 
> I still have some minor visual formatting I'd like to do (eg: make it responsive for mobile users), but I thought I'd get this conversation started as security is more important :-)

Mi still learning I need help
